.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
-python: 2.7
+dist: xenial
```

Comment:
Any reason for xenial instead of bionic?

Reply:
The [travis documentation](https://docs.travis-ci.com/user/languages/python/#python-37-and-higher) says to do it this way.

Reply:
that's good enough for me